### PR TITLE
Adds a skeleton remote RR banner

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
Adds a skeleton reader revenue banner that is served from the contributions service similar to https://github.com/guardian/dotcom-rendering/pull/1538.

To enable this we need to update emotion to version 10 - which required some small changes to the AudioPlayer component (the only usage of emotion on frontend previously).  One of the changes removes the dependency on a very old DCR package (guui) for React and replaces with a direct reference to React.

Still to come in another PR is the polyfill/shim required for ESM in older browsers.

## Screenshots
image

image

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    // Temporarily disable banner while service is developed to serve actual banner and logic
+    if(!enabled) {
+        return Promise.resolve(false);
+    }
+
+    return fetchBannerData()
+        .then(moduleData => {
+            data = moduleData;
+            return data !== null;
```
Comment (tjmw) [inline static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner.js:24]: Can you lose the `let data` defined higher up and convert this to `return moduleData !== null` (or perhaps `return !!moduleData`)? I'm not sure what purpose `data` serves (though I might well be missing something!)

**Review by tjmw — APPROVED** [review]
Nice work! This looks great.

I was going to mention testing on older browsers but I see you mentioned that in the PR description :+1:

File: static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner.js (hunk 1 above)
Comment (jfsoul) [reply static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner.js:24]: We're implementing the `Banner` type here, which looks like:
```
type Banner {
  canShow: () => Promise<boolean>,
  show: () => Promise<boolean>
}
```
So without changing that contract, we can't pass data from the output of `canShow` into `show` (like we would want to for automat decisions, where the first request gives us a decision and some data).

File: static/src/javascripts/projects/common/modules/commercial/contributions-service.js [hunk 2]
```diff
@@ -169,16 +196,81 @@ const checkResponseOk = response => {
...
+        emotionCore,
+        emotionTheming,
+        emotion,
+    };
+
+    // $FlowFixMe
+    return import(/* webpackIgnore: true */ module.url)
+        .then(bannerModule => {
+            const Banner = bannerModule.Banner;
+            const props = { ...bannerModule.props };
```
Comment (tjmw) [inline static/src/javascripts/projects/common/modules/commercial/contributions-service.js:242]: Is this the right place to be getting the props? I _think_ maybe this should be `module.props`?

Comment (jfsoul) [reply static/src/javascripts/projects/common/modules/commercial/contributions-service.js:242]: Yes I think you're right, that's a good spot!

**Pull request merged**